deploy/legacy/grafana/dashboards/clusters.json

```diff
@@ -171,7 +171,7 @@
           },
           "editorMode": "code",
           "exemplar": false,
-          "expr": "sum by (cluster) (rate(networkobservability_forward_bytes[$__rate_interval]))",
+          "expr": "sum by (cluster) (rate(networkobservability_forward_bytes{instance=~\"($Nodes)[0-9]\"}[$__rate_interval]))",
```

Comment:
@aslafy-z this has been tested, see PR description with screenshots. I intentionally removed `:` and `+` since that was not working on any dashboard.

Reply:
I acknowledge there's is a flaw in this regexp. Dots have to be escaped in the nodes variable. It can be done like with advanced variables formatting feature: https://grafana.com/docs/grafana/latest/dashboards/variables/variable-syntax/#regex. Like `$Nodes` becomes `${Nodes:regex}`.

However, I'm pretty sure there's an issue in the test.

Given that `$Nodes` is `1\.1\.1\.1.1|2\.2\.2\.2`
Given that instance label is `1.1.1.1:1234`
Given that prometheus regexps are fully anchored by default

`instance=~"(${Nodes:regex}):[0-9]+"` would match
- $0 : `1.1.1.1:1234`
- $1 : `1.1.1.1`

`instance=~"(${Nodes:regex})[0-9]"` would not match anything 

I have no installed kube prom stack to check that right now, so I can't be sure, but this regex you're using looks weird.

Reply:
So the instance label only has the node IP? not the port too (`1.1.1.1:1234`)? If so, AFAIK, a regex is needed for the filtering to work correctly.

Reply:
Hi @aslafy-z, thanks for your continuous interest in this! In your original PR you were using `internal_ip` (see [here](https://github.com/microsoft/retina/blob/f03f0ef787940c50238cc70dad082a8b9fb3e812/deploy/grafana/dashboards/clusters.json#L3730) and [this PR comment](https://github.com/microsoft/retina/pull/158/files#r1680522494))

As you correctly pointed out, `instance` includes both IP and port number, while `internal_ip` represents the IP only. That's probably one of the reasons why the dashboard did not work initially, since I started off by checking out your branch for PR #158 

For the sake of reducing the scope for this PR- and solving the immediate issue of broken node selection, broken dns and pod-level dash - I think we can open a new GH issue to implement (in a new PR) the select nodes based on names, while filtering panels by underlying IPs. Which I believe, it is probably the best resolution overall.

What do you think @huntergregory @cmergenthaler ?